src/storage.ts

```diff
@@ -1,41 +1,25 @@
-/**
- * Manages using `localStorage` and `sessionStorage`.
- *
- * Has a few advantages over the native API, including
- * - failing gracefully if storage is not available
- * - you can save and retrieve any JSONable data
- *
- * All methods are available for both `localStorage` and `sessionStorage`.
- */
+class StorageFactory {
+	#storage: Storage | undefined;
```

Comment:
```suggestion
	// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields
	#storage: Storage | undefined;
```

Reply:
The reason I think is is useful is because it's newer syntax, so while it may not be relevant forever, I personally hadn't come across it and would find it useful to know what it was called to google it (hash variable javascript is difficult to google....)